be/src/user/dto/user.dto.ts

```diff
@@ -0,0 +1,17 @@
+import { IsNotEmpty, IsEmail } from 'class-validator';
```

Comment:
I think class validator doesn't help here as gql will do the checking for us. Class-validator is used for the REST API